pkg/agent/runner.go

```diff
@@ -50,6 +50,15 @@ func NewRunnerForSpec(spec *AgentSpec) (Runner, error) {
 }
 
 func (a *agentSpecRunner) RunTask(ctx context.Context, prompt string) (AgentResult, error) {
+	debugDir := ""
+	if os.Getenv("GEVALS_DEBUG") != "" {
+		if dir, err := os.MkdirTemp("", "gevals-debug-"); err == nil {
+			debugDir = dir
+		} else {
+			fmt.Fprintf(os.Stderr, "Warning: failed to create debug directory: %v\n", err)
+		}
+	}
```

Comment:
I like the env var - can we also make this a CLI flag? i.e. `--debug` ?

Reply:
👍 done in ef6ec1d1